.travis.yml

```diff
@@ -18,17 +18,17 @@ env:
   - PUPPET_VERSION="~> 4.3.0"
 matrix:
   exclude:
-  - rvm: 2.2.3
```

Comment:
This changes only the indentation, should I ignore this file for these changes for now?